lib/plug/builder.ex

```diff
@@ -262,7 +262,7 @@ defmodule Plug.Builder do
 
       quote do
         require Logger
-        Logger.unquote(level)(unquote(message))
+        :ok = Logger.unquote(level)(unquote(message))
```

Comment:
Ouch, too late.
I'd add "Matching, to make Dialyzer happy on code executing Plug.Builder.compile/3" comment. @whatyouhide Should I make another pull request?